docs/multibrokers.md

```diff
@@ -0,0 +1,218 @@
+# Multibroker
+
+Message brokers are communication services designed to connect  multiple clients on a centralized message channel, allowing each client to send and receive messages through this central service. This  form of communication is imperative on Inspr because it allows multiple applications to connect amongst them selfs, effectively forming a group in which every message send by one party is guaranteed to reach every other registered party.
+
+Knowing what are message brokers and the central role they play on Inspr, we developed a easy way to configure new brokers on Insprd, giving the user the flexibility to choose the one that best fits their requirements.  
+
+
+
+## How to configure your broker
+
+To demonstrate how to configure your Inspr cluster with a new broker this will be a step-by-step guide to install **Kafka**, which for now is the only supported broker.
+
+1. First thing you have to do is install the broker on you cluster. For Kafka this can be done simply by following this tutorial: https://bitnami.com/stack/kafka/helm.
+
+2. Next you need to configure your Insprd Instance. This is done through a command that receives the specific broker you want to install and the required configuration of your broker's instance, which may differ from broker to broker.
+
+   1. For Kafka the configurations you'll need to attain from your Kafka deployment are: 
+
+      - Address of Kafka's bootstrap servers
+      - The auto offset reset configuration
+
+   2. With these informations you can proceed by creating the configuration file to send Insprd. This file will require two more pieces of information:
+
+      - the image for that broker's sidecar, which Inspr provides to you 
+      - the address you want that sidecar to be deployed at.
+
+   The address by default should be `http://localhost` which just indicates that that deploy should happen on the same container as the other pieces of your dApp.
+      If you followed the Helm installation for Kafka suggested above, the file you need should be exactly like this:
+
+      `kafkaConfig.yaml`:
+
+      ```yaml
+      bootstrapServers: kafka.default.svc:9092
+      autoOffsetReset: earliest
+      sidecarImage: gcr.io/insprlabs/inspr/sidecar/kafka:latest
+      sidecarAddr: "http://localhost"
+      ```
+
+      If you choose to install the broker some other way, the information may change but the **format should remain like the example above**.
+
+      Finally if you are installing any broker other than Kafka you should consult the correct file format for that broker. 
+
+   3. Once you have you configuration file all you need to do is run:
+
+      ```shell
+      insprctl cluster config kafka relative_path_to/your_file.yaml
```

Comment:
<relative_path_to> indicates that there is value to be replaced in this example, the usage of `<``>` is somewhat standard

You have even done it in the line 54 of this file:
`insprctl cluster config <broker-name> <relative_file_path>`